source/_UIAHandler.py

```diff
@@ -538,6 +538,24 @@ def _isUIAWindowHelper(self,hwnd):
 		# allow the appModule for the window to also choose if this window is bad
 		if appModule and appModule.isBadUIAWindow(hwnd):
 			return False
+		if windowClass == "NetUIHWND":
+			# This class is used for various controls in MS Office.
+			# In versions of Office older than 2016 using IAccessible avoids some issues.
+			# Problems include lack of focus reporting (#4207), strange reporting of context menu items )#9252)
+			# and not being able to report ribbon sections when they starts with an edit  field (#7067)
+			# The last problem also exiists for Office 2016, however using IAccessible for collabsed ribbons
+			# causes NVDA not to report focus changes.
+			# FIX ME!

```

Comment:
Please don't leave 'Fix me' in. Please clarify this comment to specify the things fixed, and the remaining problems. Remaining problems should should have an issue number for more in depth discussion.

You can clarify this by:
- Avoid writting "this". Above, 'this class' is ambiguous, I assume it relates to the window class?
- Start lines with 'fixes:' or 'remaining issue:'

```
'NetUIHWND is used for some controls in MS office.
IAccessible should be used in place of UIA for NetUIHWND classes in office versions before 2016:
- Fixes: 'lack of focus reporting (#4207)'
- Fixes: 'strange reporting of context menu items (#9252)'
- Fixes: 'not being able to report ribbon sections when they start with an edit field (#7067)'
- Remaining issue #XYZ: In office 2016, similar to #7067, ribbon sections that start with an edit fields are not reported. When using IAccessible for collapsed ribbons stops focus changes being reported. Testing shows that focus events are emitted but ignored.
```